apps/front/src/components/ui/formInput/formInput.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+    IconButton,
+    InputAdornment,
+    TextField,
+    TextFieldProps,
+} from '@mui/material'
+import React, { FC } from 'react'
+import { Controller, get, useFormContext } from 'react-hook-form'
+import { Visibility, VisibilityOff } from '@mui/icons-material'
+import useFormInput from '@src/components/ui/formInput/useFormInput'
+
+type FormInputProps = {
+    name: string
+    isPassword?: boolean
+} & TextFieldProps
+
+const FormInput: FC<FormInputProps> = ({ name, isPassword, ...otherProps }) => {
+    const {
+        control,
```

Comment:
Jeżeli dany element `Controller` znajduje się już context'cie FormProvidera to nie trzeba przekazywać control do niego